HandleNHCreateOrUpdateInstallationCallOrchestrator/handler.ts

```diff
@@ -60,6 +68,18 @@ export const getHandler = ({
         `INSTALLATION_ID:${installationId}|IS_TEST_USER:${isUserATestUser.value}`
       );
 
+      if (!context.df.isReplaying) {
+        telemetryClient.trackEvent({
+          name: "orchestrators.createOrUpdate.isTestUser",
+          properties: {
+            featureFlag,
+            installationId,
+            isTestUser: isUserATestUser.value
+          },
+          tagOverrides: { samplingEnabled: "false" }
+        });
```

Comment:
Don't we have the risk to flood our logs? What do you expect to achieve with such tracing?

Reply:
I expect one, and just one event for each `CreateOrUpdate`. But this PR was intended to live for few hours, not as a stable change. We did it using log, so we can close it now.